bitcash/crypto.py

```diff
@@ -16,7 +17,10 @@ def double_sha256_checksum(bytestr):
 
 
 def ripemd160_sha256(bytestr):
-    return new("ripemd160", sha256(bytestr)).digest()
+    try:
+        return new("ripemd160", sha256(bytestr)).digest()
+    except Exception:
```

Comment:
Could this be narrowed down to the very exception(s) it throws in case OpenSSL fails to produce a `ripemd160`?

Reply:
done